qutip/qip/gates.py

```diff
@@ -771,16 +773,17 @@ def qubit_clifford_group(N=None, target=0):
     S = phasegate(np.pi / 2)
     E = H * (S ** 3) * w ** 3
 
-
     for op in map(
-          # partial(reduce, mul) returns a function that takes products of its argument,
-          # by analogy to sum. Note that by analogy, sum can be written partial(reduce, add).
-          partial(reduce, mul),
-          # product(...) yields the Cartesian product of its arguments. Here, each element is
-          # a tuple (E**i, X**j, S**k) such that partial(reduce, mul) acting on the tuple
-          # yields E**i * X**j * S**k.
-          product(_powers(E, 3), _powers(X, 2), _powers(S, 4))
-        ):
+        # partial(reduce, mul) returns a function that takes products
+        # of its argument, by analogy to sum. Note that by analogy,
+        # sum can be written
+        # partial(reduce, add).
+        partial(reduce, mul),
+        # product(...) yields the Cartesian product of its arguments.
+        # Here, each element is a tuple (E**i, X**j, S**k) such that
+        # partial(reduce, mul) acting on the tuple yields E**i * X**j * S**k.
+        product(_powers(E, 3), _powers(X, 2), _powers(S, 4))
+    ):
```

Comment:
How about this ? 
```
    for op in map(partial(reduce, mul), product(_powers(E, 3),
                  _powers(X, 2), _powers(S, 4))):

        # partial(reduce, mul) returns a function that takes products
        # of its argument, by analogy to sum. Note that by analogy,
        # sum can be written
        # partial(reduce, add).

        # product(...) yields the Cartesian product of its arguments.
        # Here, each element is a tuple (E**i, X**j, S**k) such that
        # partial(reduce, mul) acting on the tuple yields E**i * X**j * S**k.
        
```

Having the comments after the for loop shouldn't make too much of a difference I guess.